plugins/tableselection/plugin.js

```diff
@@ -922,7 +922,7 @@
 			if ( CKEDITOR.tools.array.indexOf( cmds, evt.data.name ) !== -1 ) {
 				callback( editor, evt.data );
 			}
-		} );
+		}, null, null, 9 );
```

Comment:
We definitely need a comment here:
- that this listener is connected with `undo` plugin listener, and it needs a higher priority than the listener in `undo`
- to reference the issue this PR resolve

```suggestion
		// Maybe put the comment here?
		}, null, null, 9 );
```